example-dataset/data_schema.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
why this change?

Reply:
I'm adding the example data and schema so that when users install they can launch and play with cellxgene immediately without having to have a dataset ready in the correct format.